src/kanata/windows/interception.rs

```diff
@@ -65,10 +66,20 @@ impl Kanata {
                             };
                             KeyEvent { code, value }
                         }
-                        ic::Stroke::Mouse { state, rolling, .. } => {
+                        ic::Stroke::Mouse {
+                            state,
+                            rolling,
+                            flags,
+                            ..
+                        } => {
                             if mouse_to_intercept_hwids.is_some()
                                 || mouse_to_intercept_excluded_hwids.is_some()
                             {
+                                if let Some(ms_mvmt_key) = mouse_movement_key {
```

Comment:
I think this needs an `is_device_interceptable()` check, and should be duplicated to the non-interception branch as well